src/main/java/no/unit/nva/model/NullSeries.java

```diff
@@ -0,0 +1,9 @@
+package no.unit.nva.model;
+
+public class NullSeries implements Series {
```

Comment:
Deserialized it will look like this, but I guess it is okay?
`...
"series": { "type": "NullSeries" }
...
`